db/crud.go

```diff
@@ -711,8 +711,13 @@ func (db *Database) PutExistingRev(docid string, body Body, docHistory []string)
 }
 
 // Allow-conflicts=false handling
-func (db *Database) IsIllegalConflict(doc *document, parentRevID string, deleted bool) bool {
-	if db.AllowConflicts() {
+/* Truth table for AllowConflicts and noconflicts flag:
+                    Allow-conflicts=true    Allow-conflicts=false
+noconflicts=true    continue checks         continue checks
+noconflicts=false   return false            continue checks */
```

Comment:
Can you throw in a quick comment clarifying that AllowConflicts is whether or not SG allows conflicts, and 'noConflicts' is whether or not the request allows conflicts?